notification_service/settings.py

```diff
@@ -94,15 +87,19 @@
 ILMOITIN_QUEUE_NOTIFICATIONS = env("ILMOITIN_QUEUE_NOTIFICATIONS")
 
 try:
-    REVISION = subprocess.check_output(["git", "rev-parse", "--short", "HEAD"]).strip()
+    REVISION = (
+        subprocess.check_output(["git", "rev-parse", "--short", "HEAD"])
+        .strip()
+        .decode("utf-8")
+    )
 except Exception:
     REVISION = "n/a"
 
-sentry_sdk.init(
+sent = sentry_sdk.init(
     dsn=env.str("SENTRY_DSN"),
     release=REVISION,
     environment=env("SENTRY_ENVIRONMENT"),
-    integrations=[DjangoIntegration()],
+    enable_tracing=True,
```

Comment:
Do we really want to start tracing? **I think it at least should be configurable**. It will generate a lot of logs and requests. Tracing is to track how well the app performs. In Node-clients the Sentry SDK has a sample rate configuration https://docs.sentry.io/platforms/node/performance/#configure. Is there same for Python? - I could not find one. The sample rate is to reduce the tracing samples, so reducing the amount of requests.

Reply:
Earlier when I used sentry.io server for Liikunta for tracing, some one from Kanslia contacted me that the Sentry subscribtion was fully used by the Liikunta. It made more requests (because of the tracing) than all the other applications in that sentry instance together. The monthly credits ended in 2 days.

Reply:
Now I found the Python SDK config for sampling transactions: https://docs.sentry.io/platforms/python/configuration/sampling/?original_referrer=https%3A%2F%2Fwww.google.com%2F#sampling-transaction-events

Reply:
And here is what the tracing does in Django app: https://docs.sentry.io/platforms/python/integrations/django/#performance-monitoring

```
Behavior

Issue Reporting

If you use django.contrib.auth and you've set send_default_pii=True in your call to init, user data (such as current user id, email address, username) will be attached to error events.

Request data will be attached to all events: HTTP method, URL, headers, form data, JSON payloads. Sentry excludes raw bodies and multipart file uploads.

Logs emitted by any logger will be recorded as breadcrumbs by the [Logging](https://docs.sentry.io/platforms/python/integrations/logging/?original_referrer=https%3A%2F%2Fwww.google.com%2F) integration (this integration is enabled by default).

Performance Monitoring

The following parts of your Django projectRepresents your service in Sentry and allows you to scope events to a distinct application. are monitored:

* Middleware stack
* Signals
* Database queries
* Redis commands
* Access to Django caches
```

Reply:
[Added parameters for controlling Sentry tracing.](https://github.com/City-of-Helsinki/notification-service-api/pull/34/commits/1d5c750d1ec5057109393ca106c8a083b9dd01a6)